### PR TITLE
#8 Update API server for Postgres

### PR DESCRIPTION
**Pull request by jrozal**
Verified UI still works as expected

### CODE REVIEW

**Comment by jrozal** [comment]
Hello Team,
Can one of you please code review the revised API to work with postgres? Thank you!

File: server/controllers/index.js [hunk 1]
```diff
@@ -1,17 +1,18 @@
 const models = require('../models');
+const { v4: uuidv4 } = require('uuid');
 
 const getPhotos = (req, res) => {
   const room_id = req.params.id;
-  models.read({ room_id: room_id })
+  models.read(room_id)
```
Comment (danielharmon) [inline server/controllers/index.js:6]: Good use of destructuring, I've started to write my all my code around being able to do this.

File: server/models/index.js [hunk 2]
```diff
@@ -1,36 +1,34 @@
-const { Photo } = require('../../database/schema.js');
+const { Rooms, Photos } = require('../../database/postgres/models/index.js');
 
-const read = (params) => {
-  return Photo
-    .find(params)
-    .exec()
-    .then(photos => photos)
+const read = async (params) => {
```
Comment (danielharmon) [inline server/models/index.js]: since you are returning a promise does this need to be an async function?

**Review by danielharmon — APPROVED** [review]
Good work!

File: server/models/index.js (hunk 2 above)
Comment (jrozal) [reply server/models/index.js]: Thank you for catching this, Danny. I have removed the unnecessary async.

**Pull request merged**